magento/magento2
# #14971 Improper Handling of Pagination SEO

This PR removes the p= query string parameter when navigating to the first page - to avoid duplicate content SEO issues.

### Description (*)
First page pagination link include the pagination query string (for example p=1)  when on a later page - which causes a duplicate content SEO issue.

### Fixed Issues (if relevant) (*)
#14971: Improper Handling of Pagination SEO

### Manual testing scenarios (*)
* Visit a category with more than 1 page
* Use pagination to go to page 2
* Return to page 1 via pagination and verify that the pagination query string parameter is not present in the URL

### Manual testing scenarios (*)
* Visit a category with more than 1 page
* Use pagination to go to page 2
* Add a filter
* Return to page 1 via pagination and verify that the pagination query string parameter is not present in the URL, but the filter is
